$MirOS: ports/www/dillo/v2/patches/patch-dw_table_cc,v 1.2 2010/02/21 20:08:42 bsiegert Exp $
--- dw/table.cc.orig	Sun Jun  7 03:40:25 2009
+++ dw/table.cc	Mon Aug 24 16:21:42 2009
@@ -23,6 +23,7 @@
 #include "../lout/msg.h"
 #include "../lout/misc.hh"
 
+#undef MAX
 #define MAX misc::max
 
 
